package.json

```diff
@@ -30,10 +31,12 @@
   "dependencies": {
     "@modelcontextprotocol/sdk": "~1.5",
     "axios": "~1.8",
+    "dart-tools-ts": "./../dart-tools-ts",
```

Comment:
This should be updated once https://github.com/its-dart/dart-tools-ts/pull/3 is merged, and a new release will be created and published.

Reply:
Fixed